sql/builtin_functions.h

```diff
@@ -15,10 +15,14 @@
 namespace Datacratic {
 namespace MLDB {
 
+// Empty string to avoid construction of temporary object
+extern const Utf8String NO_FUNCTION_NAME;
+
 inline void checkArgsSize(size_t number, size_t expected,
-                          std::string fctName="")
+                          const Utf8String & fctName_=NO_FUNCTION_NAME)
```

Comment:
I wonder why there is a default value for this argument.  Don't we always have a function name to call this with?